mod/projects/spec/set/self/project_spec.rb

```diff
@@ -0,0 +1,8 @@
+describe Card::Set::Self::Project do
+  specify "view core" do
+    expect_view(:core).to have_tag("div.card") do
```

Comment:
this is confusing.  Why does core view have any of these classes?  It shouldn't have any of them.  Is this test actually passing??